server/common/api/admin.py

```diff
@@ -1,3 +1,9 @@
 from django.contrib import admin
 
-# Register your models here.
+from .models import *
+
+admin.site.register(User)
```

Comment:
Unless the ordering changes the behavior, any list like this should be alphabetically sorted, so it's easy to find things in the list.

Reply:
I sorted all import in alphabetical order